src/main.js

```diff
@@ -30,9 +30,12 @@ let info = information => {
 };
 
 try {
-  let yamlFilePath = core.getInput('yaml-file-path');
+  let yamlFilePath = core.getInput('yaml-file-path', {
+    required: true,
+    trimWhitespace: true
+  });
   let yamlDocData = fs.readFileSync(yamlFilePath, 'utf8', warn);
-  let output_json = core.getInput('output-json').toLowerCase() == 'true';
+  let output_json = core.getBooleanInput('output-json');
```

Comment:
Can these three variables be `const`s instead?

Reply:
I'll do one better and change all of the things to consts that need to be. :)